test/mvnormal.jl

```diff
@@ -4,6 +4,7 @@ import PDMats: ScalMat, PDiagMat, PDMat
 
 using Distributions
 using LinearAlgebra, Random, Test
+using SparseArrays
```

Comment:
@ElOceanografo this is the cause for CI failing, you need to add SparseArrays to the test in `Project.toml`